court_ball_game/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from court.models import Court
+from player.models import BallGame
+
+MAX_LEN_BALL_GAME_STR = 10
```

Comment:
This is not being used anywhere as far as i can see.

Reply:
Removed.